builtin/providers/aws/resource_aws_internet_gateway.go

```diff
@@ -50,6 +50,27 @@ func resourceAwsInternetGatewayCreate(d *schema.ResourceData, meta interface{})
 		return err
 	}
 
+	resource.Retry(5*time.Minute, func() *resource.RetryError {
+		_, err := conn.DescribeInternetGateways(&ec2.DescribeInternetGatewaysInput{
```

Comment:
just copied it from https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_api_gateway_resource.go#L132
Not sure it is useful, other than waiting for consistency in AWS